File: native-with-db/app/src/main/java/ubb/barcoaie/lab4_ma/Service/VegetableService.java
package ubb.barcoaie.lab4_ma.Service;

import android.os.Parcel;
import android.os.Parcelable;

import org.jetbrains.annotations.NotNull;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import ubb.barcoaie.lab4_ma.Model.Vegetable;

public class VegetableService implements Serializable {
    private List<Vegetable> vegetableList;

    public VegetableService() {
        this.vegetableList = new ArrayList<>();
    }

    @NotNull
    @Override
    public String toString() {
        return "VegetableService{" +
                "vegetableList=" + vegetableList +
                '}';
    }

    public List<Vegetable> getVegetableList() {
        return vegetableList;
    }

    public void setVegetableList(List<Vegetable> vegetableList) {
        this.vegetableList = vegetableList;
    }
}

File: native-with-db-server/app/src/main/java/ubb/barcoaie/lab5_ma/service/VegetableService.kt
package ubb.barcoaie.lab5_ma.service

import android.content.Context
import ubb.barcoaie.lab5_ma.repository.VegetableRepo

class VegetableService {
    private var vegetableRepo: VegetableRepo? = null
    private var hasConnection: Boolean = false

    private var context: Context? = null
}